common/source/docs/common-ac2_simple_geofence.rst

```diff
@@ -59,29 +59,68 @@ switches.  Like with the earlier fences, another fence is erected 20m
 out which will again switch the vehicle to LAND (HOLD for Rover), if it continues away from
 home.
 
+Plane Actions
+-------------
+
+If, enabled, and  the fence radius (:ref:`FENCE_RADIUS<FENCE_RADIUS>`) or altitudes,if non-zero, (:ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` and :ref:`FENCE_ALT_MIN<FENCE_ALT_MIN>`) are exceeded, Plane will execute the :ref:`FENCE_ACTION<FENCE_ACTION>` below:
+
++------------+--------------------+
++FENCE_ACTION|  Plane             +
++============+====================+
++     0      |   Report Only      +
++------------+--------------------+
++     1      | RTL                +
++------------+--------------------+
++     2      | Guided to return   +
++            | point              +
++------------+--------------------+
++     3      |  Guided to return  +
++            |  point with pilot  +
++            |  throttle control  +
++------------+--------------------+
+
+The guided return altitude during the return to the return point after breach, if this action is selected, is determined by :ref:`FENCE_RET_ALT<FENCE_RET_ALT>` . If either :ref:`FENCE_ACTION<FENCE_ACTION>` =2 or 3 is used. A fence return point **MUST** be setup or unpredictable action will occur (see video below). And the return point can be replaced by the nearest rally point as a destination, if :ref:`FENCE_RET_RALLY<FENCE_RET_RALLY>` is enabled.
+
+
 Enabling the Fence in Mission Planner
 =====================================
 
-.. image:: ../../../images/Fence_MPSetup.png
-    :target: ../_images/Fence_MPSetup.png
+To enable Fences, go to the Mission Planner full parameter list (CONFIG->Full Parameter Tree), search for items with ``FENCE_``:
+
+.. image:: ../../../images/fence_enable.png
+    :target: ../_images/fence_enable.png
 
-The Fence can be set-up by doing the following:
+Plane Parameter List (Copter/Rover have fewer)
 
--  Connect your autopilot to the Mission Planner
--  Go to the **CONFIG/GeoFence** screen
--  Enable the FENCE: :ref:`FENCE_ENABLE<FENCE_ENABLE>` =1.
+-  Set :ref:`FENCE_ACTION<FENCE_ACTION>` = to whatever you wish for a breach action. These will vary depending on vehicle type.
+-  Set :ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` = to the altitude limit you want (in meters). This is unavailable in Rover. A value of 0 disables this limit.
```

Comment:
I don't believe a distance of zero disables this limit. Disabling the `FENCE_TYPE = ALT_MAX` disables the ceiling.

Reply:
fixed